pipeline_dp/accumulator.py

```diff
@@ -0,0 +1,103 @@
+import abc
+import typing
+import pickle
+
+
+class Accumulator(abc.ABC):
+  """
+    Base class for all accumulators.
+    Accumulators are objects that encapsulate aggregations and computations of
+    differential private metrics.
+  """
+
+  @classmethod
```

Comment:
During the reviewing [PR](https://github.com/OpenMined/PipelineDP/pull/49) I've realized that it's more convenient to have merge as a standalone function.

Could you please make merge a standalone function?

Reply:
Made it a standalone function. What do you think of making it staticmethod in Accumulator?